mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,99 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
+    from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
+
+    if evaluators is None:
+        evaluator_name_list = list(_model_evaluation_registry._registry.keys())
+        if evaluator_config is not None:
```

Comment:
If `evaluator_name_list` only contains the default value, can we loosen this restriction and infer that the evaluator config applies to the default evaluator? This way, users can easily configure the default evaluator without facing an error.

Reply:
If so the the rule become complex.

I prefer simply it by:

the evaluators argument default value is 'default' (only run default evaluator)
and if user want evaluate with all evalautors, they can set `evaluators=list_evaluators()` 

What do you think :) ?

Reply:
This could be an issue if the evaluator configs overlap. If `evaluator_name_list` only has one evaluator, can we just infer that `evaluator_config` applies to that evaluator? That way, we don't have to handle this case.